R/gcloud-install.R

```diff
@@ -106,7 +106,14 @@ gcloud_install_unix <- function() {
                               paste0("--install-dir=",
                                      path.expand(dirname(gcloud_binary))))),
                           collapse = " ")
-    terminal_command <- paste(install_args, "&&", "(source", "~/.bash_profile", "||", "true)", "&&", "gcloud", "init")
+
+    if (Sys.info()["sysname"] == "Darwin") {
```

Comment:
The parentheses create a sub-shell which is why the `&&` command doesn't see the environment variables. Are we sure this doesn't also happen on Linux?

Also, if there is no `~/.bash_profile` file will this fail?

Reply:
No, I don't know... let me give it a shot in Linux.

Reply:
Yeah, same problem in Linux. Additionally, `gcloud` seems to default to `.bashrc` in Linux, not `.bash_profile`, fixing...

```bash
(source ~/.bashrc || true) && echo $PATH
/home/rstudio/bin:/home/rstudio/.local/bin::/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/home:bin/msys-ssh-1000-18:/snap/bin

source ~/.bashrc && echo $PATH
/home/rstudio/google-cloud-sdk/bin:/home/rstudio/bin:/home/rstudio/.local/bin::/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/home:bin/msys-ssh-1000-18:/snap/bin
```